wavefront/server/modules/gold_module/gold_module/controllers/image_controller.py

```
@@ -88,3 +91,68 @@ async def process_image(
+):
+    image_str = request.image
+    image_name = request.loan_id
+
+    gold_image = None
+    # Check for data URL (base64 with MIME)
+    data_url_pattern = r'^data:(image/\w+);base64,(.+)'
+    match = re.match(data_url_pattern, image_str)
+    if match:
+        try:
+            gold_image = base64.b64decode(match.group(2))
+        except Exception:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)